EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: <to be assigned>
+title: Location Metadata Standard in ERC-721 Tokens
+author:
+Ricardo Chacon (@RicardoChacon)
+Isabella Costanza (@isabellacostanza)
+Gabriela Garro (@gabygarro)
+James Jose (@jamesjosegdsey)
+David Mendes (@de-mendes)
+discussions-to: https://ethereum-magicians.org/t/request-for-feedback-standardizing-location-metadata-and-other-metadata-in-erc-721-tokens/5985
+status: Draft
+type: Informational
```

Comment:
I think this should be "Standards Track" with the "ERC" category, since it's really a standard extension for ERC-721 tokens. If you disagree, please explain why.

Reply:
The original reason why we didn't go for this was because the nature of Metadata in ERC-721 tokens is optional. Therefore, having an enforceable standard on metadata which is optional seems contradictory at best and counterproductive for anyone wanting to implement their metadata differently at its worst. Our goal is to showcase an extensible way for ERC-721 Token Metadata Interoperability and as such, we consider that the EIP lies more within the scope of a suggestion or guide. This seemed more in line with the Informational EIP. Another consideration we investigated was backwards compatibility and if there are existing ERC-721 tokens using a location metadata field, then they would have to adapt their implementations in the event of the EIP becoming final. This could lead to negatively impacting products and platforms in the community which is not in line with our values and goals of working with the community and contribute to everyone's benefit. Leaving the EIP as Informational allows for different implementations to exist if it is not in the best interest of a given party to comply with the proposed EIP.

Reply:
I understand your reasoning, but in practice this isn't really how ERCs work. Even if this EIP becomes final, it doesn't invalidate any other pre-existing projects with their own implementations, it merely means that they aren't compliant with your ERC. There is no obligation to follow an ERC, though they do help the ecosystem converge on a single way of performing tasks. As such, this should be in the `ERC` category.